.github/release.yml

```diff
@@ -0,0 +1,14 @@
+changelog:
+  exclude:
+    labels:
+      - changelog-ignore
+  categories:
+    - title: ":bug: Bug Fix"
+      labels:
+        - "*"
+      exclude:
+        labels:
+          - changelog-enhancement
```

Comment:
```suggestion
```
I don't think this is necessary? See https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#example-configuration

I'd change the order, though, so Enhancements first.

Reply:
I was keeping the previous order, but i agree.  changing

It wasnt clear from the example whether "*" would exclude only other labels that were listed before in the conf file, so i added it explicitly.  changed now